src/metabase/api/pulse.clj

```diff
@@ -42,6 +42,11 @@
   (classloader/require 'metabase-enterprise.sandbox.api.util
                        'metabase-enterprise.advanced-permissions.common))
 
+(defn- email-channel
+  "Get email channel from an alert."
+  [alert]
+  (m/find-first #(= :email (keyword (:channel_type %))) (:channels alert)))
```

Comment:
What guarantees that there will be only one?

Reply:
UI unfortunately, also the fact that we can only have one email connection per instance.

FWIW this is copied over from api/alert.clj